apps/constants/base.py

```diff
@@ -510,6 +510,8 @@
     {'name': _('Level 9'), 'points': 3000000},
 ]
 
+REVIEW_LIMITED_DELAY_HOURS = 20
```

Comment:
Yeah, we're not planning on changing this frequently. It might change once or twice, if at all.